src/main/java/pl/put/poznan/transformer/logic/text/letter_case/LowerText.java

```diff
@@ -0,0 +1,28 @@
+package pl.put.poznan.transformer.logic.text.letter_case;
+
+import pl.put.poznan.transformer.logic.text.Text;
+import pl.put.poznan.transformer.logic.text.TextTransformer;
+
+/**
+ * Lower given text
+ */
+public class LowerText extends TextTransformer {
+    public LowerText(Text transformedText){
+        super(transformedText);
+    }
+
+    @Override
+    public String transform(){
+        transformedText.transform();
+        return setLower(transformedText);
+    }
+    /**
+     * Lower given text
+     *
+     * @param transformedText holds the text
+     * @return lowered text
+     */
+    private String setLower(Text transformedText){
+        return transformedText.transform().toLowerCase();
```

Comment:
Tu się coś nie zgadza (i dotyczy to chyba wszystkich klas w tym pakiecie), niby wyżej w implementacji transform wywołujemy transform na kolejnym ze stosu, ale tutaj też. Któreś z wywołań `transformedText.transform()` jest nadmiarowe (pewno powinno zostać to wyżej, aby byc zgodnymi ze wzorcem).